scripts/example.jl

```diff
@@ -2,26 +2,27 @@
 using FreeBird
 
 # First, we need to read the walkers from a file
+# Note that the periodic boundary conditions are updated to True, True, False
 ats = read_walkers("scripts/slab-08-4x4.extxyz",pbc="TTF")
-# Note that the periodic boundary conditions are set to True, True, False
+
+# Update the system to have 80 frozen particles and 8 free particles
+ats = [AtomWalker{2}(at.configuration;list_num_par=[80,8],frozen=Bool[1,0]) for at in ats]
 
 # Then, we need to create a potential object
-lj = LJParameters(epsilon=0.1, sigma=2.5, cutoff=4.0, shift=0.5)
+lj = LJParameters(epsilon=0.1, sigma=2.5, cutoff=4.0, shift=true)
 
 # Now, we can create the liveset by combining the walkers and the potential
-ls = LJAtomWalkers(ats[1:10], lj, 80)
-# The last argument is the number of frozen particles
+ls = LJAtomWalkers(ats, lj)
 
 # We can now create a Monte Carlo object
-mc = MCDemonWalk(max_add_steps=10000, e_demon_tolerance=0.01u"eV")
```

Comment:
Do we have any tests of MC random vs. demon walks?

Reply:
Not yet. By solely doing demon walks, we can't make the NS energies to go down consistently, i.e., NS will stuck at the lowest energy from the initial set of walkers. A mixed strategy is available but is slower than pure MC random walks. Will continue working on this.

Reply:
Demon walk should be included. It works, just need some more tests/benchmarks to show how well (or bad) it works.